### PR TITLE
Scale test for global hub manager

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Ref:https://issues.redhat.com/browse/ACM-6180, https://issues.redhat.com/browse/ACM-6381

- Performance enhancement
  - https://github.com/stolostron/multicluster-global-hub/pull/632
  - https://github.com/stolostron/multicluster-global-hub/pull/652
  - https://github.com/stolostron/multicluster-global-hub/pull/660
- Postgres configuration
  - https://github.com/stolostron/multicluster-global-hub/pull/628
 - Skip python test
   - https://github.com/openshift/release/pull/43574

### CODE REVIEW

**Comment by yanmxa** [comment]
test ci/prow/test-unit

**Comment by clyang82** [comment]
Append the original thoughts
image

File: doc/simulation/README.md [hunk 1]
```diff
@@ -4,38 +4,264 @@
...
+    ```
+
+  - Create the environment so that the agent can start
+  
+    ```bash
+    # related namespace(kubectl version must be 1.24+)
+    kubectl create ns multicluster-global-hub --dry-run=client -oyaml | kubectl apply -f -
+    kubectl create ns application-system --dry-run=client -oyaml | kubectl apply -f -
+    # apply the CRDs
+    kubectl apply --server-side=true --validate=false -f pkg/testdata/crds
```
Comment (clyang82) [inline doc/simulation/README.md:38]: what is root dir here?

File: doc/simulation/README.md [hunk 2]
```diff
@@ -4,38 +4,264 @@
...
+    Deleting Simulated managedCluster managedcluster-2...
+    managedcluster.cluster.open-cluster-management.io "managedcluster-2" deleted
+    ...
+    Deleting Simulated managedCluster managedcluster-300...
+    managedcluster.cluster.open-cluster-management.io "managedcluster-300" deleted
+    ```
+
+## Mock batch policy resources on the managed clusters
+
+- Initialize the policy on the above clusters
```
Comment (clyang82) [inline doc/simulation/README.md:94]: can we consolidate the scripts to have:
1. setup.sh ( user provide global hub kubeconfig), including create 5 managed hubs with 300 managed cluster per managed hub, import the managed hubs into global hub automatically and create policies
2. start_watch.sh
3. stop_watch.sh

**Review by clyang82 — CHANGES_REQUESTED** [review]
can we move the images folder as a sub-folder of simulation?

File: doc/simulation/README.md (hunk 1 above)
Comment (yanmxa) [reply doc/simulation/README.md:38]: This is the global hub project, such as:
``` bash
$ ll
total 608
-rw-r--r--   1 yanmeng  staff   233B Mar 14  2023 CODE_OF_CONDUCT.md
-rw-r--r--   1 yanmeng  staff   222B Mar 14  2023 CONTRIBUTING.md
-rw-r--r--   1 yanmeng  staff    11K Mar 14  2023 LICENSE
-rw-r--r--@  1 yanmeng  staff   3.3K Aug  1 09:55 Makefile
-rw-r--r--@  1 yanmeng  staff   157B Aug  1 09:55 OWNERS
-rw-r--r--   1 yanmeng  staff   4.1K Sep  6 23:14 README.md
-rw-r--r--   1 yanmeng  staff   203B Mar 14  2023 SECURITY.md
drwxr-xr-x   8 yanmeng  staff   256B Sep  4 16:44 agent
drwxr-xr-x   5 yanmeng  staff   160B Sep  6 23:14 cicd-scripts
-rw-r--r--   1 yanmeng  staff   5.1K Mar 14  2023 development.md
drwxr-xr-x  19 yanmeng  staff   608B Sep 11 12:46 doc
-rw-r--r--   1 yanmeng  staff    12K Sep  6 23:14 go.mod
-rw-r--r--   1 yanmeng  staff   232K Sep  6 23:14 go.sum
drwxr-xr-x   9 yanmeng  staff   288B Sep  4 16:44 manager
drwxr-xr-x  15 yanmeng  staff   480B Sep  6 23:14 operator
drwxr-xr-x  13 yanmeng  staff   416B Aug  1 09:55 pkg
drwxr-xr-x   7 yanmeng  staff   224B Aug  1 09:55 release-notes
drwxr-xr-x   7 yanmeng  staff   224B Sep  6 23:14 samples
-rw-r--r--@  1 yanmeng  staff   485B Aug  1 09:55 sonar-project.properties
drwxr-xr-x   7 yanmeng  staff   224B Aug  1 09:55 test
drwxr-xr-x@ 17 yanmeng  staff   544B Sep  5 16:04 vendor
```

**Comment by yanmxa** [comment]
/unhold

I still need the find out why the Postgres pods need so much memory when running the test 

And we can proceed with the PR 
/cc @clyang82 @bjoydeep

**Pull request merged**